### ISSUE TITLE
[proposal] Illuminate\Support composer.json including symfony/var-dumper

### ISSUE THREAD

**Issue by underparnv** [laravel/framework#8629]
TL;DR: Include `symfony/var-dumper` in `illuminate/support`'s composer.json.

I am utilizing the `illuminate/support` package outside of Laravel. While I'm developing I am a very liberal user of the `dd()` function provided in the [helpers.php](https://github.com/laravel/framework/blob/5.0/src/Illuminate/Support/helpers.php#L443) file.

Well, at least I was.

When I upgraded to Laravel 5.0 I found that because the illuminate/support package is now utilizing `symfony/var-dumper` and isn't including it in it's composer.json dependencies, I was now getting an exception instead of pretty dumped output! Yikes! :smile: 

Therefore my proposal (and I can PR this...no big deal) is that the illuminate/support package include `"symfony/var-dumper": "2.6.*"` as a dependency.

Thoughts?

**Comment by Douglasdc3** [comment]
Your correct, symfony/var-dumper dependency should be in the illuminate/support composer. Fix proposed
